src/cloudWatchLogs/explorer/cloudWatchLogsNode.ts

```diff
@@ -18,14 +18,23 @@ import { PlaceholderNode } from '../../shared/treeview/nodes/placeholderNode'
 import { makeChildrenNodes } from '../../shared/treeview/treeNodeUtilities'
 import { LogGroupNode } from './logGroupNode'
 
-export class CloudWatchLogsNode extends AWSTreeNodeBase {
-    private readonly logGroupNodes: Map<string, LogGroupNode>
+export abstract class CloudWatchLogsParentNode extends AWSTreeNodeBase {
```

Comment:
nit: is "Base" the conventional name?
```suggestion
export abstract class CloudWatchLogsNodeBase extends AWSTreeNodeBase {
```